R/axis.R

```diff
@@ -38,4 +38,89 @@ easy_rotate_x_labels <- function(angle = 90, side = c("left", "middle", "right")
     ggplot2::theme(axis.text.x = ggplot2::element_text(angle = angle, hjust = hjust))
 }
 
+#' Easily remove axe(s)
+#'
+#' easy_remove_axes by default removes both axes, but can remove only x or y
+#' if "x" or "y" is given to the 'which' argument
+#' 
+#' easy_remove_x_axis and easy_remove_y_axis remove just the x or y axis,
+#' respectively.
+#'
+#' @md
+#' @param which which axis or axes to remove, by default "both"
+#' @param what what axis components to remove
+#' @param teach print longer equivalent ggplot expression?
+#'
+#' @return  a \code{\link[ggplot2]{theme}} object  which can be used in 
+#' \code{\link[ggplot2]{ggplot2}} calls 
+#' @export
+#' @author Alicia Schep
+#' @import rlang
+#' @import ggplot2
+#'
+#' @examples
+#' 
+#' # Remove all axes
+#' ggplot(mtcars, aes(wt, mpg)) +
+#'   geom_point() + easy_remove_axes()
+#' 
+#' # remove just x axis
+#' ggplot(mtcars, aes(wt, mpg)) +
+#'   geom_point() + easy_remove_x_axis()
+#'   
+#' # can also use:
+#' ggplot(mtcars, aes(wt, mpg)) +
+#'   geom_point() + easy_remove_axes("x")
+#'   
+#' # Remove y axis
+#' ggplot(mtcars, aes(wt, mpg)) +
+#'   geom_point() + easy_remove_y_axis()
+#'   
+#' # Remove just the ticks
+#' # Remove y axis
+#' ggplot(mtcars, aes(wt, mpg)) +
+#'   geom_point() + easy_remove_y_axis(what = "ticks)
+easy_remove_axes <- function(which = c("both","x","y"),
```

Comment:
From a code style perspective, could you please add a space after commas in argument lists? In general I think we're otherwise on the same page conventions-wise.

Reply:
Yes I will try to be better with spacing! I'm kind of a sloppy typer, should really get in the habit of using a linter that will check for these things more frequently :smile: